SILENT HILL 3 Redux.cbp

```diff
@@ -19,6 +19,9 @@
 					<Add option="-masm=intel" />
```

Comment:
Hmm, it mustn't have saved my project file, properly, as I removed that flag. I'll fix it now.